shell/platform/common/accessibility_bridge.cc

```diff
@@ -43,7 +43,29 @@ void AccessibilityBridge::AddFlutterSemanticsCustomActionUpdate(
 }
 
 void AccessibilityBridge::CommitUpdates() {
+  // AXTree cannot move a node in a single update.
+  // This must be split across two updates:
+  //
+  // * Update 1: remove nodes from their old parents.
+  // * Update 2: re-add nodes (including their children) to their new parents.
+  //
+  // First, start by removing nodes if necessary.
+  std::optional<ui::AXTreeUpdate> remove_reparented =
+      CreateRemoveReparentedNodesUpdate();
+  if (remove_reparented.has_value()) {
+    tree_.Unserialize(remove_reparented.value());
+
+    std::string error = tree_.error();
+    if (!error.empty()) {
```

Comment:
FML_DCHECK instead?

Reply:
or assert, I am not sure which is more preferred

Reply:
For now we still use assert in the desktop embedders. There's a long story behind that, but we should be consistent.

Reply:
Added an `assert(false)`. I checked `FML_UNREACHABLE` but it looks like that crashes in release builds, which is undesirable.

Reply:
it is sad that assert can't carry the message.